packages/mc-html-template/lib/load-env.js

```diff
@@ -1,38 +1,78 @@
 /* eslint-disable @typescript-eslint/explicit-function-return-type */
 const fs = require('fs');
+const replaceall = require('replaceall');
 
 // Keep a reference to the loaded config so that requiring the module
 // again will result in returning the cached value.
 let loadedEnv;
 
-const START_OF_ENV_PLACEHOLDER = 'env:';
-const isEnvVariablesPlaceholder = valueOfEnvConfig =>
-  valueOfEnvConfig && valueOfEnvConfig.startsWith(START_OF_ENV_PLACEHOLDER);
-const substitutePlaceholder = valueOfEnvConfig => {
-  const nameOfEnvVariable = valueOfEnvConfig.replace(
-    START_OF_ENV_PLACEHOLDER,
-    ''
-  );
+/**
+ * NOTE:
+ *  Allows env variable placeholders e.g. `${env:MC_API_URL}`.
+ *  Other placeholder types might be supported in the future.
+ */
+const variableSyntax = RegExp('\\${([ ~:a-zA-Z0-9._\'",\\-\\/\\(\\)]+?)}', 'g');
```

Comment:
These RegExps come from Prisma. It's actually not a standard or anyting. They've also just hardcoded stuff and this into their cli-engine.

I took the RegExps (allowing us to maybe to more in the future) while adding the matching code as much as possible.